ca/server.go

```diff
@@ -449,7 +451,7 @@ func (s *Server) Run(ctx context.Context) error {
 			if len(clusters) != 1 {
 				return errors.New("could not find cluster object")
 			}
-			s.UpdateRootCA(ctx, clusters[0]) // call once to ensure that the join tokens are always set
```

Comment:
There is one more place where we have to change this (in manager) before closing that.